docs/Reply.md

```diff
@@ -180,6 +163,19 @@ fastify.get('/', function (request, reply) {
 
 If you want to completely customize the error response, checkout [`setErrorHandler`](https://github.com/fastify/fastify/blob/error-docs/docs/Server-Methods.md#seterrorhandler) API.
 
+Errors with a `status` or `statusCode` property equal to `404` will be routed to the not found handler, checkout
+the [`server.setNotFoundHandler`](https://github.com/fastify/fastify/blob/error-docs/docs/Server-Methods.md#setnotfoundhandler) API to know how to modify that:
```

Comment:
"handler. See the [`server.setNotFoundHandler`] API to learn more about handling such cases:"